### DISCUSSION TITLE
나는 어떤 개발자가 되고 싶은가

### DISCUSSION THREAD

**Discussion by giscus** [Cyma-s/Cyma-s.github.io#91] [General]
# 나는 어떤 개발자가 되고 싶은가

태양 가까이 날면 위험해, 이카로스. 넌 앞으로 뭘 하고 싶어? 지금은 부끄럽지만, 나에게는 그런 자부심이 있었다. 내가 좋아하는 것만큼은 잘할 수 있다. 하지만 좋아하는 것을 찾는 것이 더 어려운 일이었다. 어렸을 때부터 항상 주위의 누군가 물어보던 "넌 앞으로 뭘 하고 싶어?" 라는 질문에 대답하는 게 가장 힘들었으니 말이다. 그래서 그런 갈증이 있었…

https://vero.wiki/level4-writing/

**Comment by greeng00se** [comment]
날개를 고쳐 매는 베로위키 응원합니다.

**Reply by Cyma-s** [reply]
허브는 강철 날개잖아요
